resources/migrations/001_update_migrations.yaml

```
@@ -5824,6 +5824,89 @@ databaseChangeLog:
             sql: >-
               DROP VIEW IF EXISTS v_content;
 
+  - changeSet:
+      id: v50.2024-02-29T15:06:43
+      author: tsmacdonald
+      comment: Add the field_usage join table
+      changes:
+        - createTable:
+            tableName: field_usage
+            remarks: Fields used by a card
+            columns:
+              - column:
+                  name: id
+                  remarks: PK
+                  type: int
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: card_id
+                  remarks: referenced card
+                  type: int
+                  constraints:
+                    nullable: false
+                    referencedTableName: report_card
+                    referencedColumnNames: id
+                    foreignKeyName: fk_field_usage_card_id
+                    deleteCascade: true
+              - column:
+                  name: field_id
+                  remarks: referenced field
+                  type: int
+                  constraints:
+                    nullable: false
+                    referencedTableName: metabase_field
+                    referencedColumnNames: id
+                    foreignKeyName: fk_field_usage_field_id
+                    deleteCascade: true
+              - column:
+                  remarks: Name of the used column, as it is in the query
+                  name: column_name
+                  type: varchar(254)
+                  constraints:
+                    nullable: false
+              - column:
+                  remarks: Name of the used table, as it is in the query
+                  name: table_name
+                  type: varchar(254)
+                  constraints:
+                    nullable: false
+              - column:
+                  remarks: Is this field usage still valid?
+                  name: is_currently_valid
```

```suggestion
                  name: valid
```
nit: I think we usually just use an adjective for it, like report_card.archived or metabase_field.active

---

upon further reading, this is just a duplication of `metabase_field.active`, is there a reason why we need to keep track of it in `field_usage`? 

the only place I find it's used is to find stale queries, which you already joined with metabase_field